deploy/clientcomm.tf

```diff
@@ -0,0 +1,86 @@
+// This is a terraform definition for an AWS configuration of clientcomm.
+// See the README.md for more details on how to use this.
+
```

Comment:
all the below seems pretty straightforward, but could you add brief comments to explain what each step does?